.github/workflows/tests.yml

```diff
@@ -5,7 +5,7 @@ on: [push, pull_request]
 jobs:
   autotests:
     runs-on: ubuntu-latest
-    container: python:3.8
+    container: python:3.9
```

Comment:
It is better to use github matrix here for testing with different versions of python(3.8, 3.9)
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix